wled00/e131.cpp

```diff
@@ -135,9 +138,8 @@ void handleE131Packet(e131_packet_t* p, IPAddress clientIP, byte protocol){
     availDMXLen = (dmxChannels - DMXAddress) + dmxLenOffset;
   }
 
-  // DMX data in Art-Net packet starts at index 0, for E1.31 at index 1
-  if (protocol == P_ARTNET && dataOffset > 0) {
-    dataOffset--;
+  if (dataOffset > 0) {
```

Comment:
The next line should be guarded with `#ifdef WLED_ENABLE_DMX`, to avoid compile errors in "normal" builds  without full DMX support.